src/rohiniBDec19/Assignment2/Program4.java

```diff
@@ -0,0 +1,33 @@
+package rohiniBDec19.Assignment2;
+
+/*4) Largest and smallest number in unsorted array? */
+
+
+public class Program4 {
+	
+	int temp;
+	void findNum(int arr[]) {	
+		for(int i=0;i<arr.length;i++) {
+			for(int j=i+1;j<arr.length;j++)
+			{
+				if(arr[i]>arr[j])
+				{
```

Comment:
Cool @RohiniBDec19 
going good, keep it up

Reply:
Thanks Harsh